__tests__/hooks/useGroupInfo.test.js

```diff
@@ -412,28 +452,32 @@ describe('useGroupInfo should run correctly', () => {
     expect(spyGallery).toHaveBeenCalled();
   });
 
-  it('uploadImageBase64 should run correctly', async () => {
+  it('uploadImage should run correctly', async () => {
     const navigation = {
       push: jest.fn(),
       navigate: jest.fn()
     };
-    const spyService = jest.spyOn(serviceFile, 'uploadFile');
+    const spyService = jest
+      .spyOn(ImageUtils, 'uploadImage')
+      .mockImplementation(() => ({data: {url: 'https://detik.jpg'}}));
     const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper});
-    await result.current.uploadImageBase64({base64: '1234'});
+    await result.current.uploadImage('file:///imag.jpg');
     expect(spyService).toHaveBeenCalled();
     expect(result.current.uploadedImage).toEqual('https://detik.jpg');
     expect(result.current.isUploadingImage).toBeFalsy();
   });
 
-  it('error uploadImageBase64 should run correctly', async () => {
+  it('error uploadImage should run correctly', async () => {
     const navigation = {
       push: jest.fn(),
       navigate: jest.fn()
     };
-    const spyService = jest.spyOn(serviceFile, 'uploadFile');
+    const spyService = jest
+      .spyOn(ImageUtils, 'uploadImage')
+      .mockImplementation(() => ({data: {url: 'https://detik.jpg'}}));
     const spyConsole = jest.spyOn(console, 'log');
     const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper: wrapperError});
-    await result.current.uploadImageBase64({base64: '1234'});
+    await result.current.uploadImage('file:///imag.jpg');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `uploadImage` function is now being tested instead of the `uploadImageBase64` function. This change seems to be part of a larger refactoring effort where the image upload functionality has been moved to the `ImageUtils` module. The test checks that the `uploadImage` function is called and that it updates the state correctly.
```diff
-  it('uploadImageBase64 should run correctly', async () => {
+  it('uploadImage should run correctly', async () => {
     const navigation = {
       push: jest.fn(),
       navigate: jest.fn()
     };
-    const spyService = jest.spyOn(serviceFile, 'uploadFile');
+    const spyService = jest
+      .spyOn(ImageUtils, 'uploadImage')
+      .mockImplementation(() => ({data: {url: 'https://detik.jpg'}}));
     const {result} = renderHook(() => useGroupInfo({navigation}), {wrapper});
-    await result.current.uploadImageBase64({base64: '1234'});
+    await result.current.uploadImage('file:///imag.jpg');
     expect(spyService).toHaveBeenCalled();
     expect(result.current.uploadedImage).toEqual('https://detik.jpg');
     expect(result.current.isUploadingImage).toBeFalsy();
   });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->